test/unit/lbp-manager-factory.spec.js

```diff
@@ -167,46 +167,6 @@ describe(">> Contract: LBPManagerFactory", () => {
   });
   context("» deploy LBP using LBPManager with wrong values", () => {
     let params;
-    it("$ reverts on invalid swapFeePercentage", async () => {
```

Comment:
The problem here is that we can set SwapFee too low(or high) while initializing LBPManager, but we won't be able to change it when we'll want to create the LBP from LBPManager. The contract will be useless after this error, because swapFee is impossible to change. So it makes sense to put a require statement in the `initializeLBPManager` function. Or as a possibility add function to change `swapFeePercentage`.

Reply:
I think, according to the point you're making we should also check for the length of the tokenList/amounts/startWeights/endWeights arrays because we might have the same issue there?

Reply:
Nope, we won't, because we initialize the arrays in the `initialiseLBP`, and they are limited to the length of 2, even if we'll use longer arrays. It won't be bad decision, but it will eat up a bit of gas.
![image](https://user-images.githubusercontent.com/31989062/136374296-469b9f29-cd5f-4d65-a808-b4bbb1ff3751.png)